tests/fixtures/fixtures_tasking.py

```diff
@@ -23,23 +23,23 @@ def tasking_mock(auth_mock, requests_mock):
         json_data_product_schema = json.load(json_file)
         requests_mock.get(url=url_data_product_schema, json=json_data_product_schema)
 
-    url_get_quotations_mp1 = f"{constants.API_HOST}{QUOTATION_ENDPOINT}?page=0&sort=createdAt,desc"
+    url_get_quotations_mp1 = f"{constants.API_HOST}{QUOTATION_ENDPOINT}?page=0&sort=createdAt,desc"  # noqa: E231
```

Comment:
To suppress flake8's warning:
`E231 missing whitespace after ','`

Same for other places as well.

Reply:
E231 doesn't apply to all these cases, I run flake8 without those changes on fresh installation without any complaints. Please revert those changes